### PR TITLE
Eng 13833 - Backport HTTP session capability to v7.7.2

### PR DESCRIPTION
**Pull request by GuaGuaDean**

### CODE REVIEW

**Comment by manjujames** [comment]
Are the protobuf jar changes part of this fix? Do we need that change?

File: src/frontend/org/voltdb/utils/ApiRequestServlet.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import javax.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.server.Request;
+
+import org.voltdb.VoltDB;
+import org.voltdb.client.ClientResponse;
+
+/**
+ *
+ * @author akhanzode
+=======
```
Comment (manjujames) [inline src/frontend/org/voltdb/utils/ApiRequestServlet.java]: We should get rid of these merge artifacts.

**Comment by manjujames** [comment]
What are the last 4 jar changes for (zk jars, commons and httpclient jar updates)?

File: src/frontend/org/voltdb/utils/ApiRequestServlet.java (hunk 1 above)
Comment (GuaGuaDean) [reply src/frontend/org/voltdb/utils/ApiRequestServlet.java]: Sure. 
For the protobuf jar, Anish made this change in his pro commit's classpath file. https://github.com/VoltDB/pro/commit/dd259789bdce4585f64bf39a4b971b96c066304f
ZK, commons.io and httpclient used for test only and were introduced by earlier commits

**Pull request merged**